### ISSUE TITLE
Saved maps missing parts & meshes

### ISSUE THREAD

**Issue by KoyanskayaEggnog** [Roblox-Devs/magnus-night-v3#9]
Hello o/ Whenever a map is saved (With and without terrain) I notice there are missing bricks or meshes when loaded up in studio. Hopefully these pictures can help out

First picture is of the studio / loaded version, Second would be the regular game to show what parts are missing. 
![image_2024-01-12_160034077](https://github.com/Roblox-Devs/magnus-night-v3/assets/156469006/5c4afdb7-3677-42bb-9fa9-100c15144efc)

![image_2024-01-12_155958030](https://github.com/Roblox-Devs/magnus-night-v3/assets/156469006/5e0b0f50-c21a-4eb4-8acb-fee2eca52190)

Take care!

**Comment by stan2474** [comment]
send game link

**Comment by KoyanskayaEggnog** [comment]
> send game link

Here you go!

[](https://www.roblox.com/games/5741630507/Akisa-Shore-SHOWCASE)

**Comment by KoyanskayaEggnog** [comment]
https://www.roblox.com/games/5741630507/Akisa-Shore-SHOWCASE

**Comment by Eetblgb25** [comment]
i dont think this issue is fixable since there is no union data getting saved.
